sdk/appconfiguration/azure-appconfiguration/azure/appconfiguration/__init__.py

```diff
@@ -19,4 +24,9 @@
     "AzureAppConfigurationClient",
     "ConfigurationSetting",
     "ResourceReadOnlyError",
+    "FeatureFlagConfigurationSetting",
+    "SecretReferenceConfigurationSetting",
+    "PERCENTAGE",
```

Comment:
Should we have a enum for these constants?

Reply:
Anna/Johan recommended a constant string rather than an Enum.